compiler/il/OMRNode_inlines.hpp

```diff
@@ -176,7 +176,7 @@ OMR::Node::mayUse()
 TR_UseDefAliasSetInterface
 OMR::Node::mayKill(bool gcSafe)
    {
-   if (self()->getOpCode().hasSymbolReference() && (self()->getOpCode().isLikeDef() || self()->mightHaveVolatileSymbolReference())) //we want the old behavior in these cases
+   if (self()->getOpCode().hasSymbolReference() && (self()->getOpCode().isLikeDef() || self()->mightHaveNonTransparentSymbolReference())) //we want the old behavior in these cases
```

Comment:
Again, I'm wondering whether `OpaqueSemantics` really needs to be treated conservatively here, and if this could safely check for "at least or stronger than acquire/release".  Does this need to worry only about what other threads might do?  If so, I don't think the fact that opaque operations need to be performed atomically would matter.